src/com/aca/narekhakobyan/homework/items/models/PenRecord.java

```diff
@@ -0,0 +1,117 @@
+package com.aca.narekhakobyan.homework.items.models;
+
+import com.aca.narekhakobyan.homework.items.concepts.MemoryGadget;
+import com.aca.narekhakobyan.homework.items.functionality.Voice;
+import com.aca.narekhakobyan.homework.items.functionality.format.FormatSound;
+
+import java.io.*;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Objects;
+import java.util.logging.FileHandler;
+import java.util.logging.Logger;
+
+public class PenRecord extends MemoryGadget implements Voice {
+    private final FormatSound formatSound;
+    private final String name = "Pen Record";
+    private final PenColor penColor;
+
+    public PenRecord(FormatSound formatSound, Integer memory, Integer recDuration, Integer price, Integer count, PenColor penColor) {
+        super(memory, recDuration, price, count);
+        this.formatSound = formatSound;
+        this.penColor = penColor;
+    }
+
+    public static PenRecord initFromFile() {
```

Comment:
move this logics related to reading from files into some DataConverter class, and make your models independent from your initialization ways